### PR TITLE
Update GetCertificatePolicy-example.json

### PR DESCRIPTION
**Pull request by sebansal**
adding basic constraint description

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.0/examples/GetCertificatePolicy-example.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
           "key_usage": [],
           "validity_months": 297,
           "basic_constraints": {
-            "ca": false
+            "ca": false,
+            "description": "Optional for leaf certs but needs to be true for CA certificate, if building PKI on top of key vault."
```
Comment (heaths) [inline specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.0/examples/GetCertificatePolicy-example.json:26]: Should capitalize "Key Vault" when used as a product name:
```suggestion
            "description": "Optional for leaf certs but needs to be true for CA certificate, if building PKI on top of Key Vault."
```

**Review by heaths — APPROVED** [review]

File: specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2-preview/examples/GetCertificatePolicy-example.json [hunk 2]
```diff
@@ -21,7 +21,11 @@
           "subject": "CN=KeyVaultTest",
           "ekus": [],
           "key_usage": [],
-          "validity_months": 297
+          "validity_months": 297,
+          "basic_constraints": {
+            "ca": false,
+            "description": "Optional for leaf certs but needs to be true for CA certificate, if building PKI on top of Key Vault."
```
Comment (mkreidenweis) [inline specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2-preview/examples/GetCertificatePolicy-example.json:27]: The `description` doesn't look right to me in here -- this is an example response, it wouldn't contain the description. 

The `description` should instead be added to the specification file `certificates.json`, shouldn't it?

**Comment by ravbhatnagar** [comment]
No activity for a long time. please re-add ARM feedback label if you need a review from ARM. removing to reduce noise.

File: specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2-preview/examples/GetCertificatePolicy-example.json (hunk 2 above)
Comment (heaths) [reply specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2-preview/examples/GetCertificatePolicy-example.json:27]: Additionally, the `basic_constraints` property would need to be added to the swagger. Even if this were valid example JSON, there's no representation in swagger.

**Comment by JackTn** [comment]
Hi, @sebansal. The PR has be closed for a long time and it's related branch still exist. If no further used for over 14 days, I will delete this related branch. Please tell me if you still need this branch.

**Pull request closed without merging**